packages/ckeditor5-block-quote/docs/features/block-quote.md

```diff
@@ -16,9 +16,11 @@ Use the editor below to see the block quote plugin in action. You can also prece
 
 {@snippet features/block-quote}
 
-## Nesting block quotes
+## Nested block quotes
 
-Starting from version 27.1.0, CKEditor 5 supports nesting a block quote in another block quote. This may prove useful for scientific or academic papers, but also articles citing sources and referring to previous writing would often use it. To use this functionality, simply click where you need the quote nested and insert it just like a regular one.
+Starting from version 27.1.0, CKEditor 5 will properly display a block quote nested in another block quote. The functionality itself may prove useful for scientific or academic papers, but also articles citing sources and referring to previous writing would often use it.
+
+This is a backwards compatibility issue for content created in CKEditor 4. Pasting content with nested block quotes is supported. It is also possible to nest a block quote into another block quote using the {@link features/drag-drop drag and drop} mechanism - select an existing blockquote and drag it into another existing blockquote.
```

Comment:
```suggestion
Support for nested block quotes is provided as backward compatibility for loading pre-existing content, for example created in CKEditor 4. Additionally, pasting content with nested block quotes is possible, too. You can also nest a block quote in another block quote using the {@link features/drag-drop drag and drop} mechanism &mdash; just select an existing block quote and drag it into another.
```

Using both "block quote" and "blockquote" in one sentence seems pretty extravagant to me :stuck\_out\_tongue\_winking\_eye: